src/Tes.Runner.Test/NetworkUtilityTests.cs

```diff
@@ -10,16 +10,20 @@ namespace Tes.Runner.Test
     [TestCategory("Unit")]
     public class NetworkUtilityTests
     {
-        private const string testUrl = "https://www.example.com";
+        private const string testUrl = "https://github.com";
+        private const string ruleChain = "OUTPUT";
         private readonly NetworkUtility utility = new NetworkUtility();
 
         [TestMethod]
         [TestCategory("Unit")]
+        [TestCategory("RequiresRoot")]
         public async Task BlockAndUnblockAnIpAddressOnLinux()
         {
             if (!OperatingSystem.IsLinux())
             {
                 // Not implemented; TES only supports Linux VMs
+                Console.WriteLine("Test did not run because host is not Linux.");
+                Assert.Inconclusive();
                 return;
             }
 
```

Comment:
I think you can simply remove the catch block here. You are asserting that no exception must thrown which is what tests implicity do - exceptions fail the test cases. To make it clearer you could name test case along these lines: 
GetStringAsync_IsCalledTwice_SucceedsAndNoExceptionIsThrown